docs/docs/recipes/interact-with-management-api/README.md

```diff
@@ -2,11 +2,27 @@
 sidebar_position: 6
 ---
 
+import logtoManagementApiResourceSrc from './assets/logto-management-api-resource.png';
+import logtoManagementApiDetailsSrc from './assets/logto-management-api-details.png';
+
+import BasicsAboutAccessTokenRequest from '../../../quick-starts/generic/machine-to-machine/fragments/\_basics-about-access-token-request.mdx';
+import FetchAccessTokenForLogtoManagementApi from '../../../quick-starts/generic/machine-to-machine/fragments/\_fetch-access-token-for-logto-management-api.mdx';
+import AccessTokenUsage from '../../../quick-starts/generic/machine-to-machine/fragments/\_access-token-usage.mdx';
+import AccessLogtoManagementApiUsingAccessToken from '../../../quick-starts/generic/machine-to-machine/fragments/\_access-logto-management-api-using-access-token.mdx';
+import M2mAccessTokenNote from '../../../quick-starts/generic/machine-to-machine/fragments/\_m2m-access-token-sub-note.mdx';
+import M2mRoleAssignment from '../../../quick-starts/generic/machine-to-machine/fragments/\_m2m-role-assignment.mdx';
+
 # 🚝 Interact with Management API
 
 ## What is Logto Management API
 
-The Logto Management API is a comprehensive collection of APIs that empower administrators to manage identity-related tasks, enforce security policies, and comply with regulations and standards.
+The Logto Management API is a comprehensive set of APIs that gives developers full control over their implementation to suit their product needs and tech stack. It is pre-built, listed in the API resource list, and cannot be deleted or modified.
```

Comment:
Listed in the API resource list? Do you mean the API docs?

Reply:
updated, "listed in API resource list in the Logto Console"